package.json

```diff
@@ -33,9 +32,11 @@
     "build:test:js": "rollup -c scripts/test.rollup.config.js",
     "build:test:manifest": "node -e \"var b=require('./scripts/manifest-data.js'); b.build();\"",
     "build:test:segments": "node -e \"var b=require('./scripts/segments-data.js'); b.build();\"",
-    "watch": "run-p watch:js watch:test:manifest watch:test:segments watch:test:js watch:js:switcher",
+    "prewatch": "npm run clean",
+    "watch": "run-p watch:webworker watch:js watch:test:manifest watch:test:segments watch:test:js",
```

Comment:
switcher is currently broken, with tba/lhls, going to add an issue to get it working again, but we don't need to build/rebuild something (takes ~5s) that is broken.